### PR TITLE
Adding ability to push data across XbeeTX with N.io

### PR DESCRIPTION
**Pull request by killian441**
Added a data property to XBee to allow data to be input to the XBeeTX block and sent over Xbee. The old code used json.dumps to encode the data, but this added extra quotes around the data. The current scheme seems to work well to send any arbitrary data.

### CODE REVIEW

File: xbee_at_command_block.py [hunk 1]
```diff
@@ -1,10 +1,14 @@
 import binascii
 from nio.util.discovery import discoverable
-from nio.properties import Property
+from nio.properties import Property, ListProperty, PropertyHolder
 from nio.properties.version import VersionProperty
 from .xbee_base import XBeeBase
 
 
+class ATCommand(PropertyHolder):
+    command   = Property(title='AT Command Name (ascii)', default='ID')
```
Comment (hansmosh) [inline xbee_at_command_block.py]: Take a look at PEP-8 and update the code accordingly. Remove additional spaces here. Also rename variable to not use camel case.

https://www.python.org/dev/peps/pep-0008/

**Comment by hansmosh** [comment]
@killian441: Can you please split the PRs? I want to merge the XBeeTX change but I'm not sure about the AT commands yet.

**Comment by killian441** [comment]
Yeah, sorry. I made the PR before I commited the AT changes, I didn't realize that it would include them. I thought PRs were for a specific commit.

**Comment by killian441** [comment]
@hansmosh I updated the PR and moved the AT command changes to a separate branch.

**Pull request merged**